lsp-verilog.el

```diff
@@ -0,0 +1,78 @@
+;;; lsp-verilog.el --- Verilog Client settings         -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2019 Patrick Grogan
+
+;; Author: Patrick Grogan <pogrogan@gmail.com>
+;; Created: 7 December 2019
+;; Keywords: languages, lsp, verilog
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; LSP client support for Verilog. Right now, the only supported LSP server
+;; is HDL CHecker. See https://github.com/suoto/hdl_checker
+;;
+;; This file is based on the lsp-vhdl.el file.
```

Comment:
lsp-vhdl is a bit more complex since it supports 2 servers. I think that you may use a simpler client like `lsp-crystal` defined in `lsp-clients.el`. Let me know if you have more questions.

Reply:
You suggestion is to do it like LSP-crystal or leave it as is and change the file based on in the header comment? Not sure to follow here.

Reply:
I meant that you may use crystal client as a guide and structure it in a similar manner. Let me know if you have any questions/concerns.